app/dashboard/views.py

```diff
@@ -827,13 +847,12 @@ def users_fetch(request):
             profile_json['avatar_url'] = user_avatar.avatar_url
         count_work_completed = Activity.objects.filter(profile=user, activity_type='work_done').count()
         count_work_in_progress = Activity.objects.filter(profile=user, activity_type='start_work').count()
-        previously_worked_with = 0
-        if profile:
+        if current_user:
             previously_worked_with = BountyFulfillment.objects.filter(
-                bounty__bounty_owner_github_username__iexact=profile.handle,
+                bounty__bounty_owner_github_username__iexact=current_user.username,
```

Comment:
Why use the username here instead of handle? If we are expecting a github username, then I think handle should be used, since non-github users could have username that is someone else's github handle (theoretically, at least...)
https://github.com/gitcoinco/web/blob/master/app/dashboard/models.py#L2352

Reply:
Because the built-in User model in django has only username and no handle column, hence.

Reply:
@SaptakS I understand that, but we can easily access the profile from the `current_user` object, right? So why not use the handle? Is this a performance tradeoff?